test/start.test.ts

```
@@ -445,265 +494,276 @@ describe('test/start.test.js', () => {
+      let app: Coffee;
 
       beforeEach(async () => {
-        await utils.cleanup(fixturePath);
+        await cleanup(fixturePath);
       });
 
       beforeEach(async () => {
         app && app.proc && app.proc.kill('SIGTERM');
-        await utils.cleanup(fixturePath);
+        await cleanup(fixturePath);
       });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)